test_files/augment/angular-augment.d.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * This test is simulating what it's like when someone augments a module.
+ * The interesting output is the generated externs.
+ */
+
+import * as angular from './angular';
+
+declare module './angular' {
+  /**
+   * sub is a namespace that exists in angular already; this file is
+   * augmenting it.
+   */
+  namespace sub {
+    type AugmentSubType = string;
+  }
+
+  /**
+   * sub is a new namespace introduced by this file.
+   */
+  namespace local {
+    type LocalType = string;
+  }
```

Comment:
I wonder if an simple `export let x: string;` might also be useful here. Augmentation is not just about namespaces.

Also is type only namespaces intentional, test will have greater coverage if it has some values in those too.